showcase/src/assets/translations/nl.json

```diff
@@ -87,6 +87,10 @@
       "LOGOUT": {
         "LOGOUT_DEFAULT_ICON": "Logout-knop - Standaard pictogram",
         "LOGOUT_CUSTOM_ICON": "Logout-knop - Aangepast pictogram"
+      },
+      "SIDEBAR": {
+        "TITLE": "App Sidebar",
+        "INTRO": "The preview of this code is the actual sidebar of the showcase"
```

Comment:
Could you please translate this text ?
According to Google Translate, it should look like:
`De preview van deze code is de eigenlijke zijbalk van de showcase`